### ISSUE TITLE
TypeError __LOGGER__({ label: "foo" }) is not a function

### ISSUE THREAD

**Issue by svassr** [redfin/react-server#738]
I tried to use label for a logger like follow 

`var logger = logging.getLogger(__LOGGER__({ label: "foo" }));`

I received that error about `__LOGGER__` : 

`TypeError: (intermediate value)(intermediate value) is not a function`

I'm using node v6.2.2 and the following modules
```
└── react@0.14.8
└── react-server@0.4.8 
└─┬ react-server-cli@0.4.9
  └── webpack@1.13.2 
```

**Comment by gigabo** [comment]
Thanks for reporting @svassr.  Are you using the React Server CLI to run your server?

**Comment by svassr** [comment]
Yes I am. I added the version up here

**Comment by gigabo** [comment]
Ugh, yeah this is pretty broken. 😬 

The `__LOGGER__({ label: "foo" })` syntax is supported by `react-server-module-tagger`, but not the `babel-plugin-react-server` wrapper that the CLI uses.  Totally unimplemented!

**Comment by gigabo** [comment]
Oops, looks like this got auto-closed by #778, but it's not quite resolved yet.

The `label` key is getting [passed through](https://github.com/redfin/react-server/blob/master/packages/babel-plugin-react-server/test/fixtures/label/expected.js#L7) but not [appended to the `name`](https://github.com/redfin/react-server/blob/master/packages/react-server-module-tagger/index.js#L62-L64).

There's some more discussion [here](https://github.com/redfin/react-server/pull/778#discussion_r90706006).
